capstone/src/main/resources/static/event.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <title> Create an Event </title>
+    <script src="script.js"> </script>
+    <link rel="stylesheet" href="event.css">
+  </head>
+  
+  <body>
+
+    <div class="menu">
+      <nav>
+        <a href="map.html"> Campus Map </a>
+      </nav>
+    </div>    
+
+    <h1>Create an Event </h1>
+
+    <div class="event-form">
+      <form id="form" method="POST">
+
+        <label for="organizationID"> OrganizationID: </label>
```

Comment:
If the field is hidden, you don't need a label for it. Unless you meant to display it as read-only? Same question for University field.

Reply:
I removed the labels, I'll add them later after we decide how we want to display it